samples/Azure.ResourceManager.Sample/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -118,7 +118,7 @@ Page<AvailabilitySet> NextPageFunc(string nextLink, int? pageSizeHint)
             );
         }
 
-        /// <summary> Filters the list of AvailabilitySets for a Azure.ResourceManager.Core.SubscriptionOperations represented as generic resources. </summary>
+        /// <summary> Filters the list of AvailabilitySets for a <see cref="SubscriptionOperations" /> represented as generic resources. </summary>
```

Comment:
@ShivangiReja , is this the result you wanted?

Reply:
Yes, this looks good. Thank you @AlexanderSher!